config/models.php

```diff
@@ -25,7 +25,9 @@
         |
         */
 
-        'path' => app_path('Models'),
```

Comment:
But why? In Laravel app models are stored and created by default exactly in `app` directory, not in `app/Models`.
Maybe it would be more correctly to keep Laravel defaults in default models config?